index.html

```diff
@@ -6901,6 +6907,10 @@ <h4>JsonLdErrorCode</h4>
 <section class="appendix informative" id="changes-from-cr">
   <h2>Changes since Candidate Release of 12 December 2019</h2>
   <ul>
+    <li>Added missing keyworkds <a href="#ctd-invalid-entries">28</a> of the
+      <a href="#create-term-definition">Create Term Definition algorithm</a>
+      that remaining steps are skipped if the value of `@id` is `null`.
```

Comment:
This line is a copy & paste from below.  I think it should just be removed and add period above?

Reply:
Sorry, I'll commit them to master.

Reply:
Looks like a rebase failure.